theano/gof/op.py

```diff
@@ -714,7 +738,7 @@ def R_op(self, inputs, eval_points):
             "own op, implement the R_op method." %
             (self, self.__class__.__name__))
 
-    def perform(self, node, inputs, output_storage):
+    def perform(self, node, inputs, output_storage, params=None):
```

Comment:
@abergeron I'm sorry, not able to recollect why i added this parameter. 
cc : @nouiz